feat: Added the SimpleCalc, Counter, TubePlayer and ToDo apps to the ReadME (#645)
unoplatform/Uno.Samples: README.md

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)

### SimpleCalculator App

> The SimpleCalculator App is a sample application that is designed to perform basic arithmetic operations. The app was built using 4 options - [C# Markup](https://aka.platform.uno/csharp-markup) or [XAML]() for the UI and [MVUX](https://aka.platform.uno/mvux) or [MVVM]() for state management.

Reviewer: Missing links for `[XAML]()` and `[MVVM]()`?

Reviewer: For these links I would point directly to the variants in the workshop tutorial docs:

<img alt="image" src="https://github.com/unoplatform/Uno.Samples/assets/4793020/243e0d94-2ecc-4e42-b622-89833950ea4f">

I'm not sure we have aka links created for these though @agneszitte

Reviewer 2: @vatsashah45, @kazo0, @nickrandolph
I think we could use this simple aka link (https://aka.platform.uno/simplecalc-workshop) that gives the overview of the four variants of the SimpleCalc workshop.

So we can mention something additional like:
`The sample app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).`

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc)

### Counter App

> Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.

Reviewer 2: @vatsashah45, @kazo0, @nickrandolph 
I think we could use this simple aka link (https://aka.platform.uno/counter-tutorial) that gives the overview of the four variants of the Counter workshop.

So we can mention something additional like:
`This sample app was built using the four variants of the [Counter workshop](https://aka.platform.uno/counter-tutorial), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).`

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter)

### ToDo App

Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)

### TubePlayer App

> The TubePlayer App is a sample application that leverages Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers to create a rich, responsive video streaming service.

Reviewer 2: @vatsashah45, @kazo0, @nickrandolph 
I think we could use this simple aka link (https://aka.platform.uno/tubeplayer-workshop) that gives the overview of the Tube Player workshop.

So we can mention something additional like:
`This sample app was built following the [Tube Player worshop](https://aka.platform.uno/tubeplayer-workshop).`

> The TubePlayer  App is a sample application that leverages the Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers looking to understand how to use Uno.Extensions to create a rich, responsive video streaming service.

Reviewer: ```suggestion
The TubePlayer App is a sample application that leverages Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers looking to create a rich, responsive video streaming service.
```

> The TubePlayer  App is a sample application that leverages the Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers looking to understand how to use Uno.Extensions to create a rich, responsive video streaming service.

Reviewer: `to create a rich, responsive video streaming service.`

I'm not sure that TubePlayer's UI is responsive, is it?

Reviewer: @agneszitte did we change TubePlayer to be responsive? I thought it was locked to a fixed width or something like that?

Reviewer 2: @kazo0 yes still locked to a fixed width. You are right we need to remove the "responsive" part here as that is not an aspect we are showcasing with this sample.

@kazo0, @nickrandolph 
I think the summary of this sample should be closer to some details we can find in the workshop overview, right ?
![image](https://github.com/unoplatform/Uno.Samples/assets/16295702/6b83994d-094d-4080-be13-62681c92587e)

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Commerce)

> ### SimpleCalculator App

Reviewer 2: ```suggestion
### Simple Calc App
```
To match the workshop and name of the sample app

Reviewer 3: Also try to update any subsequent uses of _SimpleCalculator_

> The SimpleCalc App is a sample application that is designed to perform basic arithmetic operations. The app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).

Reviewer 4: ```suggestion
The SimpleCalc App is a sample application designed to perform basic arithmetic operations. This app was built using the four variants of the [Simple Calc workshop](https://aka.platform.uno/simplecalc-workshop), combining different markup languages (XAML or C# Markup) and presentation frameworks (MVVM or MVUX).
```

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/SimpleCalc)

### Counter App

> Discover the simplicity and power of Uno.Extensions with the Counter App, a straightforward yet powerful demonstration of basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and UI updates in real-time.

Reviewer 4: ```suggestion
Experience the simplicity and power of Uno.Extensions through the Counter App, a straightforward yet powerful demonstration of both basic and advanced features of the Uno Platform. This app provides a hands-on experience with fundamental concepts such as state management, user interaction, and real-time UI updates.
```

This sample app was built using the four variants of the [Counter workshop](https://aka.platform.uno/counter-tutorial), combining markup language (XAML or C# Markup) and presentation framework (MVVM or MVUX).

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/Counter)

### ToDo App

> Dive into the essentials of task management with the ToDo App, a meticulously designed sample application that showcases the power and flexibility of Uno.Extensions. By focusing on creating and managing a to-do list, this app demonstrates practical applications of core Uno.Extensions features, offering a hands-on experience in developing responsive, user-friendly interfaces across platforms.

Reviewer 4: ```suggestion
Dive into the essentials of task management with the ToDo App, a meticulously crafted sample application that highlights the power and flexibility of Uno.Extensions. By emphasizing the creation and organization of to-do lists, this app showcases practical applications of essential Uno.Extensions features, offering a hands-on experience in crafting responsive and user-friendly interfaces across multiple platforms.
```

[Browse source](https://github.com/unoplatform/Uno.Samples/tree/master/reference/ToDo)

### TubePlayer App

> The TubePlayer App is a sample application that leverages Uno Platform to recreate the core functionalities of YouTube. This app serves as a comprehensive tutorial for developers to create a rich video streaming service.

Reviewer 2: @kazo0, @nickrandolph do you think "The TubePlayer App is a sample application that leverages Uno Platform **to recreate the core functionalities of YouTube**" isn't a bit much here? 

As it is just a search and stream video experience.
![image](https://github.com/unoplatform/Uno.Samples/assets/16295702/4ae4f08f-a77e-4099-a836-6a365ec36b7f)

> The TubePlayer App is a sample application that enables the user to search for, and stream, Youtube videos. This app was made using tools, libraries, and patterns available in the Uno Platform, that are there to help rapidly build high-quality applications.

Reviewer 4: ```suggestion
The TubePlayer App is a sample application that allows users to search for, and stream YouTube videos. This app was created using the tools, libraries, and patterns provided by the Uno Platform, designed to facilitate the rapid development of high-quality applications.
```

> This sample app was built following the [Tube Player worshop](https://aka.platform.uno/tubeplayer-workshop).

Reviewer 4: ```suggestion
This sample app was built following the [Tube Player workshop](https://aka.platform.uno/tubeplayer-workshop).
```
